tagging_tools/md_index.cc

```diff
@@ -35,30 +37,55 @@ using namespace policy_engine;
 #include <string>
 #include <cstring>
 
-bool extract_metadata_values(std::string file_name, reporter_t *err) {
-  metadata_memory_map_t metadata_memory_map; 
+void usage() {
+  printf("usage: md_index <tag_file> <policy_dir>\n");
+}
+
+
+int main(int argc, char **argv) {
+  stdio_reporter_t err;
+  const char *tag_filename;
+  const char *policy_dir;
+  metadata_memory_map_t metadata_memory_map;
```

Comment:
I think a comment explaining the relationship between the maps, index_maps, and the metadata_values would be helpful here.